libs/barista-components/sunburst-chart/src/sunburst-chart.util.ts

```diff
@@ -50,8 +64,8 @@ export interface DtSunburstNodeInternal {
   showLabel: boolean;
 }
 
-export interface DtSunburstSlice
-  extends d3.PieArcDatum<DtSunburstNodeInternal> {
+export interface DtSunburstChartSlice
```

Comment:
This should not be part of the public api

Reply:
It's exported from util to be used in component but it's not exported in index.ts

Reply:
Yeah right now it's fine - previously the select method was public that needed that type so it was part of the public API i guess ;) Now it is perfectly fine :)